### PR TITLE
Added labels for the unused levels in Prepare>Column: Factor>Remove Unused Levels dialog

### LINKED ISSUE #7024: No label for the unused levels control on Remove Unused Levels Dialogue?
Is there any reason why there is no label/tooltip for this control?

image

### PR DESCRIPTION
**Pull request by EstherNjeriLiberatta**
Fixes #7024
@N-thony @africanmathsinitiative/developers This is ready for review.

### CODE REVIEW

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern if you test, then we can merge, thanks

**Review by rdstern — COMMENTED** [review]
I can't quite see what's different now?  Here it is.  I sort of expected a label in front of the green control below?

**Review by rdstern — DISMISSED** [review]
I can't quite see what's different?  
image

I sort of expected a label like `Unused Levels:` in front of the green control.  But I don't think it is vital, so I am approving.

**Comment by lloyddewit** [comment]
@N-thony Thank you for raising issue #7024 . The issue refers to the 'unused levels control' but this could refer either to the control @EstherNjeriLiberatta labelled, or the control that @rdstern is referring to.
Which of these controls (or both?) did you want labelled?
Thanks for clarifying

**Comment by N-thony** [comment]
> @N-thony Thank you for raising issue #7024 . The issue refers to the 'unused levels control' but this could refer either to the control @EstherNjeriLiberatta labelled, or the control that @rdstern is referring to. Which of these controls (or both?) did you want labelled? Thanks for clarifying

@lloyddewit that one too, if it is okay with @EstherNjeriLiberatta, it will be good to add it i.e `Unused Levels:` in front of the green control?

**Comment by EstherNjeriLiberatta** [comment]
@N-thony @africanmathsinitiative/developers this is ready for review

**Review by N-thony — APPROVED** [review]

File: instat/dlgRemoveUnusedLevels.Designer.vb [hunk 1]
```diff
@@ -94,28 +96,48 @@ Partial Class dlgRemoveUnusedLevels
...
+        Me.ucrInputUnusedLevels.TabIndex = 6
+        '
+        'lblLevelsPreview
+        '
+        Me.lblLevelsPreview.AutoSize = True
+        Me.lblLevelsPreview.Location = New System.Drawing.Point(239, 56)
+        Me.lblLevelsPreview.Name = "lblLevelsPreview"
+        Me.lblLevelsPreview.Size = New System.Drawing.Size(81, 13)
+        Me.lblLevelsPreview.TabIndex = 3
+        Me.lblLevelsPreview.Text = "Levels preview:"
```
Comment (lilyclements) [inline instat/dlgRemoveUnusedLevels.Designer.vb:122]: Given that the other labels is capital for both words, can this change to "Levels Preview:"?

**Review by lilyclements — COMMENTED** [review]
Very very minor comment, otherwise looks good to me!

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern This is an uncontroversial change and it now has 2 approvals (thank you @lilyclements ) so I'll go ahead and merge. I hope that's OK.

**Comment by rdstern** [comment]
Great

**Pull request merged**